.github/workflows/broken_links_checker.yaml

```diff
@@ -0,0 +1,21 @@
+name: Check for Broken Links in Markdown
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - '*'
```

Comment:
Probably overkill to run on every push, but may be worth changing this to run on any PR that is trying to merge to main. 🤷